sdk/storage/Azure.Storage.Files.Shares/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/61b7e4ad91d44d3dacfcaee6c5f4cfd008022a76/specification/storage/data-plane/Microsoft.FileStorage/preview/2022-11-02/file.json
+    - https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/97114bc1a965387eea64e1470c85b8f44590223f/specification/storage/data-plane/Microsoft.FileStorage/preview/2022-11-02/file.json
```

Comment:
TODO update this when https://github.com/Azure/azure-rest-api-specs/pull/22699 merges